site.mk

```diff
@@ -1,29 +1,29 @@
+GLUON_MULTIDOMAIN=1
+
+GLUON_FEATURES := \
+	respondd \
+	autoupdater \
+	web-wizard \
+	ebtables-limit-arp \
+	ebtables-filter-multicast \
+	ebtables-filter-ra-dhcp \
+	web-admin \
+	web-autoupdater \
+	web-network \
+	web-wifi-config \
+	mesh-batman-adv-15 \
+	mesh-vpn-fastd \
+	radvd \
+	status-page \
+	config-mode-domain-select \
+	ebtables-filter-arp-ffmuc \
```

Comment:
this should be obsoleted by now by gluons own features.

Reply:
[This](https://github.com/freifunkMUC/gluon-packages/blob/master/gluon-ebtables-filter-arp-ffmuc/files/lib/gluon/ebtables/109-mcast-drop-non-site) looks like it is specific for ffmuc. Should it be skipped either way?

Reply:
please, read the release notes starting with... v2016.2 i guess.
this package does the same and more, configured via site.conf: https://github.com/freifunk-gluon/gluon/tree/master/package/gluon-ebtables-source-filter
https://github.com/freifunk-gluon/gluon/blob/master/docs/package/gluon-ebtables-source-filter.rst